src/c++/devices/tracer/dmTracer.c

```diff
@@ -19,6 +19,18 @@
 #include "common.h"
 #include "traceLoggerBlockTrace.h"
 
+#ifndef VDO_UPSTREAM
+#undef VDO_USE_ALTERNATE
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(10, 0))
+#define VDO_USE_ALTERNATE
+#endif
+#else /* !RHEL_RELEASE_CODE */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 10, 0))
```

Comment:
I don't think this is correct. Functions blk_limits_io_{min,opt} exist for the 6.10 and 6.11 kernels. It's 6.12 that they do not exist for. Seems like this should be if LINUX_VERSION_CODE < KERNEL_VERSION(6, 12, 0).